assets/sass/okta-theme.scss

```diff
@@ -58,7 +58,7 @@
 
 
 // scss-lint:disable IdSelector
-#okta-sign-in {
+#okta-sign-in.auth-container {
```

Comment:
I made the theme more specific. The argument is that themes should be able to override the base at any point.